source/Stargate-API-Skeleton/StargateApplication.class.st

```diff
@@ -126,7 +155,7 @@ StargateApplication >> applicationInfoConfiguration [
 { #category : #'private - accessing' }
 StargateApplication >> authAlgorithm [
 
-	^ JWAHMACSHA256 parameterValue
+	^ 'HS256'
```

Comment:
May not be such a bad idea to write a comment in this method explaining where this string comes from.

Reply:
Same for JWTBearerAuthenticationFilterTest >> algorithmName